packages/core/src/components/overflow-list/overflow-list.md

```diff
@@ -1,6 +1,6 @@
 @# Overflow list
 
-this is cool
+`OverflowList` displays a list of items that can overflow on its own. The required `visibleItemRenderer` callback prop allows for customizing the appearance of visible items. The required `overflowRenderer` callback prop allows for customizing the rendering of collapsed items. 
```

Comment:
"on its own" means nothing. we should clarify what causes overflowing -- specifically, resize.